test/torchaudio_unittest/backend/sox_io/load_test.py

```diff
@@ -319,72 +293,92 @@ def test_amr_nb(self):
         self.assert_format("amr-nb", sample_rate=8000, num_channels=1, bit_depth=32, duration=1)
 
 
-@skipIfNoExec("sox")
 @skipIfNoSox
 class TestLoadParams(TempDirMixin, PytorchTestCase):
     """Test the correctness of frame parameters of `sox_io_backend.load`"""
 
-    original = None
-    path = None
+    def _test(self, func, frame_offset, num_frames, channels_first, normalize):
+        original = get_wav_data("int16", num_channels=2, normalize=False)
```

Comment:
The `normalize` argument in `get_wav_data` is how the original data is generated, while the `normalize` argument in the test is about what value to pass to the load function, so that data is converted to float32 on-the-fly, no matter what the original data type is. (note that the WAV format stores data with int16. There is almost no application other than DL that handles float32 WAV files)

To test the `normalize` argument of `load` function, we need to test on waveform other than float32, (int16 is the common choice) so that we can check switching `normalize` of `load` function would change the resulting Tensor dtype. If the original dtype is float32, we do not know if `normalize` argument of `load` is properly working.